website/src/controllers/groups.js

```diff
@@ -211,7 +211,6 @@ api.create = function(req, res, next) {
       if (err == 'Already in a party, try refreshing.') return res.json(400,{err:err});
       if (err) return next(err);
       return res.json(populated);
-      group = user = null;
```

Comment:
Ahh, good catch. Not :100: sure doing that will actually help, but I'll put them back before the return just in case.